chaos-monkey-spring-boot/src/test/java/de/codecentric/spring/boot/chaos/monkey/endpoints/ChaosMonkeyRequestScopeRestEndpointIntTest.java

```diff
@@ -53,7 +53,7 @@
     classes = ChaosDemoApplication.class,
     webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
 @TestPropertySource("classpath:application-test-chaos-monkey-profile.properties")
-class ChaosMonkeyRequestScopeRestEndpointIntegration {
+class ChaosMonkeyRequestScopeRestEndpointIntTest {
```

Comment:
I would stick here with the longer name "...IntegrationTest" instead of "...IntTest"

Reply:
Done